lib/forest_liana.rb

```diff
@@ -20,15 +20,21 @@ module UserSpace
   mattr_accessor :excluded_models
   mattr_accessor :included_models
   mattr_accessor :user_class_name
+  mattr_accessor :names_overriden
 
   self.apimap = []
   self.allowed_users = []
   self.models = []
   self.excluded_models = []
   self.included_models = []
   self.user_class_name = nil
+  self.names_overriden = {}
 
   def self.schema_for_resource resource
     self.apimap.find {|collection| collection.name == resource.table_name}
```

Comment:
This method is broken. A simple search on Movies Customers (for example) crashes.